python/paddle/fluid/contrib/slim/tests/qat_int8_comparison.py

```diff
@@ -164,12 +168,20 @@ def _predict(self,
                      model_path, exe, 'model', 'params')
 
             graph = IrGraph(core.Graph(inference_program.desc), for_test=True)
+            graph.draw('.', 'qat_orig', graph.all_op_nodes())
             if (transform_to_int8):
-                mkldnn_int8_pass = TransformForMkldnnPass(
-                    scope=inference_scope, place=place)
-                mkldnn_int8_pass.apply(graph)
+                if (test_case_args.qat2):
+                    transform_to_fp32_pass = TransformThroughFP32Pass(
+                        scope=inference_scope, place=place, core=core)
+                    graph = transform_to_fp32_pass.apply(graph)
+                else:
+                    mkldnn_int8_pass = TransformForMkldnnPass(
```

Comment:
Same as above, after enabling the performance full stack optimization, we should remove TransformForMkldnnPass this temporary solution.

Reply:
Done.